sql/updates/world/master/2024_99_99_99_world.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE `creature_template` 
+CHANGE COLUMN `entry` `CreatureID` INT UNSIGNED NOT NULL DEFAULT 0 FIRST,
+CHANGE COLUMN `name` `Name` MEDIUMTEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL AFTER `KillCredit2`,
+CHANGE COLUMN `femaleName` `NameAlt` MEDIUMTEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL AFTER `Name`,
```

Comment:
Why do we pick a short string here and have verbose names everywhere else?

name2 or nameAlternative. Maybe name1 and name2? That would actually make querying easier since `name` is a reserved word in MySQL, no need to escape anymore.